tests/unit-tests/AuthTests.swift

```diff
@@ -525,7 +525,9 @@ class AuthTests: XCTestCase {
         XCTAssertNil(internalAPI.auth.authToken)
     }
     
-    func testAuthTokenRefreshRetryOnlyOnce() {
+    func testAuthTokenRefreshRetryOnlyOnce() throws {
+        throw XCTSkip("skipping this test")
```

Comment:
Why do we throw with this message? Maybe I just don't understand the usage

Reply:
Might wanna include a comment as to why we're skipping if this test intentionally, otherwise another developer wouldn't have context as to why it was skipped